PendingReleaseNotes

```diff
@@ -101,6 +101,11 @@
   `PutBucketPolicy`. Additionally, the root user will always have access to modify
   the bucket policy, even if the current policy explicitly denies access.
 
+* CephFS: The "ceph fs volume create" command now allows users to pass metadata
```

Comment:
Actual command in double backticks please, not a doublequote character.

Reply:
Fixed.q

Reply:
Fixed.